src/main/java/me/micartey/webhookly/embeds/EmbedObject.java

```diff
@@ -25,4 +30,20 @@ public class EmbedObject {
 
     private final List<Field> fields = new ArrayList<>();
 
+    public void setTimestampFromOffsetDateTime(OffsetDateTime offsetDateTime) {
+        timestamp = offsetDateTime.toString();
+    }
+
+    public void setTimestampFromTemporalAccessor(TemporalAccessor temporalAccessor) {
+        if (temporalAccessor instanceof Instant) {
+            timestamp = OffsetDateTime.ofInstant((Instant) temporalAccessor, Clock.systemDefaultZone().getZone()).toString();
+        } else {
+            timestamp = OffsetDateTime.from(temporalAccessor).toString();
+        }
+    }
+
+    public void setTimestampFromTimeInMillis(long timeInMillis) {
+        setTimestampFromTemporalAccessor(Instant.ofEpochMilli(timeInMillis));
+    }
+
```

Comment:
Preferably, it shouldn't be required to add these methods.
Use the builder instead.

Reply:
All right, I will remove them.